### ISSUE TITLE
Opera extension

### ISSUE THREAD

**Issue by hartungstenio** [OneNoteDev/WebClipper#301]
Opera and Chrome share the same browser engine, and very similar extension architeture.

It shouldn't be hard to make na Opera extension.

**Comment by leakymemory** [comment]
I agree that generating a Opera-specific extension wouldn't be hard.  We actually talked about doing this a few months back, but ultimately decided against it because the Opera user base is fairly small and we were hesitant to add another platform to our list (as there is a bit of a cost each time we release a new version).

Also, I don't know if you are aware of this or not, but there is actually [an extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/?display=en) you can install in Opera which will allow you to install any of the Chrome extensions.  Through this you can install the [OneNote WebClipper on Chrome](https://chrome.google.com/webstore/detail/onenote-web-clipper/gojbdfnpnhogfdgjbigejoaolejmgdhk) today. :-)

Having said all that, we'll keep this on the list and talk about when the right time to release something officially might be.  The fact that you are asking for it is a good indication that we should at least consider adding it to the build.

Thanks!
